pkg/cassandra/config_tag.go

```diff
@@ -41,57 +46,91 @@ func (p cassConfigTagPath) isInline() bool {
 
 // parseCassConfigTag parses a textual representation of a cass-config tag.
 // The textual representation is of the following general form:
-// tag := <path1>[,<path2>,...,<pathN>][;<option1>;<option2>;...;<optionN>]
-// path := <constraint>:[<segments>]
+// tag := <path1>[;<path2>;...;<pathN>][;<option1>;<option2>;...;<optionN>]
+// path := <typeAndConstraint1>[,<typeAndConstraint2>,...,<typeAndConstraintN>]:[<segments>]
+// typeAndConstraint := [<type>@]<constraint>
+// type := any string (defaults to cassandra if not present)
```

Comment:
Is `type` by itself used any more? From the changes in `parsePath` it doesn't look like it.

Reply:
It is still used to build the map of version constraints by server type.